pkg/clipboard/clipboard_test.go

```diff
@@ -16,8 +16,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestCopyToClipboard(t *testing.T) {
+func TestNotExistingClipboardCopyCommand(t *testing.T) {
 	_ = os.Setenv("GOPASS_NO_NOTIFY", "true")
+	_ = os.Setenv("GOPASS_CLIPBOARD_COPY_CMD", "not_existing_command")
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	buf := &bytes.Buffer{}
+	out.Stdout = buf
+
+	assert.Error(t, CopyTo(ctx, "foo", []byte("bar"), 1))
+}
+
+func TestUnsupportedCopyToClipboard(t *testing.T) {
+	_ = os.Setenv("GOPASS_NO_NOTIFY", "true")
+	_ = os.Setenv("GOPASS_CLIPBOARD_COPY_CMD", "")
```

Comment:
We have a nice [helper](https://github.com/gopasspw/gopass/blob/master/internal/action/clone_test.go#L121) to temporarily unset env vars and restore them later. Maybe we should have that also for setting vars in tests.

Reply:
Introduced in bfee59e9